### PR TITLE
Change email for team joined + test

### PR DESCRIPTION
**Pull request by gannetson**

### CODE REVIEW

**Review by eodolphi — APPROVED** [review]

File: bluebottle/activities/triggers.py [hunk 1]
```diff
@@ -266,22 +287,45 @@ class TeamTriggers(TriggerManager):
...
+                #     ParticipantStateMachine.remove
+                # ),
+                RelatedTransitionEffect(
+                    'activity',
+                    TimeBasedStateMachine.reopen,
+                    conditions=[team_activity_will_not_be_full]
+                ),
+                # TeamContributionTransitionEffect(ContributionStateMachine.fail),
+                # NotificationEffect(TeamCancelledMessage),
+                # NotificationEffect(TeamCancelledTeamCaptainMessage)
```
Comment (eodolphi) [inline bluebottle/activities/triggers.py:313]: Just remove stuff it is no longer needed

File: bluebottle/test/test_runner.py [hunk 2]
```diff
@@ -11,7 +12,7 @@
 
 class MultiTenantRunner(DiscoverSlowestTestsRunner, InitProjectDataMixin):
     def setup_databases(self, *args, **kwargs):
-
+        self.keepdb = getattr(settings, 'KEEPDB', self.keepdb)
```
Comment (eodolphi) [inline bluebottle/test/test_runner.py:15]: why is this?

**Review by eodolphi — CHANGES_REQUESTED** [review]

File: bluebottle/activities/triggers.py (hunk 1 above)
Comment (gannetson) [reply bluebottle/activities/triggers.py]: It _was_ needed :-) Comment it out for testing purposes. Now it's back

File: bluebottle/test/test_runner.py (hunk 2 above)
Comment (gannetson) [reply bluebottle/test/test_runner.py:15]: For local IDE testing stuff...

**Pull request merged**